include/core/parameters_cfd_dem.h

```diff
@@ -43,6 +43,49 @@ namespace Parameters
     gidaspow
   };
 
+
+  /**
+   * @enum DragCoupling
+   * @brief Defines the numerical coupling strategy used for drag force computation between fluid and particles.
+   *
+   * This enumeration specifies how the drag interaction term is treated in time
+   * integration schemes for coupled CFD-DEM or multiphase simulations. The
+   * choice of coupling affects both stability and computational cost. The
+   * fully_implicit and fully_explicit schemes are named this way since explicit
+   * is a reserved keyword in C++. Thus, we use fully_implicit and
+   * fully_explicit to ensure a coherent terminology.
+   *
+   * - **fully_implicit**:
+   *   The drag force is evaluated using the current (implicit) fluid
+   *   velocities and previous (explicit) particle velocity. The resulting
+   *   drag is applied to both the particles and the fluid.
+   *   Provides maximum stability, but requires more Newton iterations.
+   *
+   * - **semi_implicit**:
+   *  The drag force on the particles is computed using the previous velocities
+   * of both the particles and the fluid. When transferring momentum to the
+   * fluid, the previous fluid velocity is used to evaluate the momentum
+   * transfer coefficient, while the current fluid velocity is used to calculate
+   * relative velocity between the particle and the fluid. This provides
+   * extensive stability, but for larger time-step may lead to a slight
+   * violation of Newton's third law since the drag applied on the fluid is not
+   * strictly equal to the drag applied to the particles.
```

Comment:
Maybe I would add this full explanation to the parameter guide. At least for the semi-implicit.